docs/balsamic_map.md

```diff
@@ -1,43 +1,49 @@
-| clinical-vcf-pass-tnscope-index, snv, vcf-pass-tnscope         | False       | tnscope, vcf-snv-clinical-index     | deliver, scout        |
-| ascat-pdf, clinical-ascat-pdf                                  | False       | ascatngs, visualization             | deliver, scout        |
-| clinical-ascat-copynumber, ascat-copynumber                    | False       | ascatngs, metrics                   | deliver               |
-| vcf-vardict, research-vcf-vardict                              | False       | vardict, vcf-snv-research-index     | deliver               |
-| snv, vcf-pass-vardict, clinical-vcf-pass-vardict               | False       | vardict, vcf-snv-clinical           | deliver, scout        |
-| snv, vcf-pass-vardict, clinical-vcf-pass-vardict-index         | False       | vardict, vcf-snv-clinical-index     | deliver, scout        |
-| vcf-tnscope-umi, research-vcf-tnscope-umi                      | False       | tnscope-umi, vcf-umi-research       | deliver               |
-| research-vcf-tnscope-umi-index, vcf-tnscope-umi                | False       | tnscope-umi, vcf-umi-research-index | deliver               |
-| snv, vcf-pass-tnscope-umi, research-vcf-pass-tnscope-umi       | False       | tnscope-umi, vcf-umi-clinical       | deliver, scout        |
-| snv, vcf-pass-tnscope-umi, research-vcf-pass-tnscope-umi-index | False       | tnscope-umi, vcf-umi-clinical-index | deliver, scout        |
-| cns, cnv-cns                                                   | False       | cnvkit, metrics, segments           | deliver               |
-| scatter, cnv-scatter                                           | False       | cnvkit, visualization, scatter      | deliver               |
-| diagram, cnv-diagram                                           | False       | cnvkit, visualization, diagram      | deliver               |
-| cnv-gene-metrics, gene-metrics                                 | False       | cnvkit, metrics, genes              | deliver               |
-| cgh-file, cnv-somatic-cgh-file                                 | False       | cnvkit, vcf2cytosure                | deliver, scout        |
-| clinical-rd-delly, rd-delly                                    | False       | delly, metrics                      | deliver               |
+| Balsamic tags                                                  | Mandatory   | HK tags                                 | Used by               |
+|----------------------------------------------------------------|-------------|-----------------------------------------|-----------------------|
+| balsamic-config                                                | True        | balsamic-config                         | audit, cg             |
+| balsamic-report                                                | True        | balsamic-report                         | audit                 |
```

Comment:
Is this the delivery report? If so, will it be also uploaded to scout and deliver?

Reply:
No, the delivery report generation, storage and delivery logics are in CG. This is just the snakemake html report located in `/analysis/delivery_report/` that we are creating with (`balsamic report deliver ...`).